rmgpy/data/kinetics/rules.py

```diff
@@ -705,7 +705,10 @@ def estimateKinetics(self, template, degeneracy=1):
         if degeneracy > 1:
             kinetics.comment += "\n"
             kinetics.comment += "Multiplied by reaction path degeneracy {0}".format(degeneracy)
-
+        
+        kinetics.comment += "\n"
+        kinetics.comment += "family: {0}".format(self.label.replace('/rules',''))
```

Comment:
I know, but it isn't in the kinetics comments in the seed so I needed to add it.